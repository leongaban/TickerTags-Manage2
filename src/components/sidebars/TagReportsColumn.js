import React from 'react'

class TagReportsColumn extends React.Component {
	constructor(props) {
		super(props)
        this.state = {};
	}

	render() {
		return (
			<section className="tag-reports">
				<h1>Tag Reports</h1>
				<div>
					<table>
						<thead></thead>
						<tbody></tbody>
					</table>
				</div>
			</section>
		)
	}
}

export default TagReportsColumn