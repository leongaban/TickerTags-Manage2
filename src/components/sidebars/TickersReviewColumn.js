import React from 'react'

class TickersReviewColumn extends React.Component {
	constructor(props) {
		super(props)
        this.state = {};
	}

	render() {
		return (
			<section className="tickers-review">
				<h1>Tickers Review</h1>
				<div>
					<table>
						<thead></thead>
						<tbody></tbody>
					</table>
				</div>
			</section>
		)
	}
}

export default TickersReviewColumn