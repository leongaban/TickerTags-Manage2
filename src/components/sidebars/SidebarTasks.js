import React from 'react'
import PropTypes from 'prop-types'
import TagReportsColumn from './TagReportsColumn'
import TickersReviewColumn from './TickersReviewColumn'

export default function SidebarTasks(props) {
return (
	<section id="sidebar-right">
		<TagReportsColumn />
		<TickersReviewColumn />
	</section>
	)
}