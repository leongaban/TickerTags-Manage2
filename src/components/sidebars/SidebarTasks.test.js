import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import SidebarTasks from './SidebarTasks'
import TagReportsColumn from './TagReportsColumn'
import TickersReviewColumn from './TickersReviewColumn'

const sidebarTasksShallow = shallow(
	<SidebarTasks/>
);

describe('<SidebarTasks /> component', () => {
	it('should render', () => {
		const tree = toJson(SidebarTasks);
		expect(tree).toMatchSnapshot();
	});

	it('should contain the TagReportsColumn component', () => {
		expect(sidebarTasksShallow.find(TagReportsColumn).length).toBe(1);
	});

	it('should contain the TickersReviewColumn component', () => {
		expect(sidebarTasksShallow.find(TickersReviewColumn).length).toBe(1);
	});

	// it('should render the Notification component if state.error is true', () => {
	// 	loginComponent.setState({ error: true });
	// 	expect(loginComponent.find(Notification).length).toBe(1);
	// });
});