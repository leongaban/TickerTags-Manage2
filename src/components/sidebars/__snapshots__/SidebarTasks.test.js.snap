// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SidebarTasks /> component should render 1`] = `undefined`;
